packages/pds/src/image/index.ts

```diff
@@ -15,10 +14,8 @@ export async function maybeGetInfo(
     ])
     metadata = result
   } catch (err) {
-    if (errHasMsg(err, 'Input buffer contains unsupported image format')) {
-      return null
-    }
-    throw err
+    // The buffer has a corrupted image or no image at all.
+    return null
```

Comment:
This is not a proper fix as it will also mute any `pipeline(stream, processor)` error.